src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -920,20 +920,47 @@ def test_storage_account_revoke_delegation_keys(self, resource_group, storage_ac
 
 @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-04-01')
 class BlobServicePropertiesTests(StorageScenarioMixin, ScenarioTest):
-    @ResourceGroupPreparer()
-    @StorageAccountPreparer(kind="StorageV2")
-    def test_storage_account_update_change_feed(self):
-        result = self.cmd('storage account blob-service-properties update --enable-change-feed true -n {sa} -g {rg}').get_output_in_json()
-        self.assertEqual(result['changeFeed']['enabled'], True)
+    @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-06-01')
+    @ResourceGroupPreparer(name_prefix='cli_storage_account_update_change_feed')
+    @StorageAccountPreparer(kind='StorageV2', name_prefix='clitest', location="eastus2euap")
```

Comment:
prefer using production region like `eastus2`

Reply:
`eastus2` is tested to not support `changeFeed` for now.

Reply:
I see.  So please make sure it is in preview